backends/redis.js

```diff
@@ -5,9 +5,12 @@ module.exports = function(Model, options) {
   if (options.ttl != null && options.ttl) {
     // Assuming the Model connector is "redis" here but we may need a config in
     // the future.
+
     Model.observe('after save', function(ctx, next) {
       // @see http://redis.io/commands/expire.
-      // TODO
+      var redisClient = ctx.Model.getConnector().client._client;
```

Comment:
I believe `client` has proxy to all the `_client` methods. See https://github.com/strongloop/loopback-connector-redis/blob/master/lib/redis.js#L71. Please use that if possible.